cmd/internal/http/handlers.go

```diff
@@ -29,9 +29,9 @@ import (
 	"github.com/google/cadvisor/validate"
 
 	auth "github.com/abbot/go-http-auth"
+	"github.com/kubernetes/utils/clock"
```

Comment:
you should use "k8s.io/utils/clock" here and elsewhere

Reply:
Yes, I have just, finally, fixed it. Sorry for generating the mess.

Reply:
`FakeClock` is not available in "k8s.io/utils/clock" - it seems that depending on import path there are different types available in the package.

Reply:
Its in k8s.io/utils/testing

Reply:
sorry, k8s.io/utils/clock/testing